react/components/molecules/key-value/key-value.js

```diff
@@ -65,8 +65,8 @@ export class KeyValue extends mix(PureComponent).with(IdentifiableMixin) {
             <Touchable
                 style={this._style()}
                 disabled={!this.props.pressable}
-                onPress={this.props.onPress}
-                onLongPress={this.props.onLongPress}
+                onPress={() => this.props.onPress(this.props._key, this.props.value)}
+                onLongPress={() => this.props.onLongPress(this.props._key, this.props.value)}

```

Comment:
Isn't the key passed as prop? Why does it need to be in the callback?

Reply:
I figured it would be better to pass it in the callback when dealing with multiple key-values as it will allow us to globally handle key value presses like this: https://github.com/ripe-tech/ripe-robin-revamp/pull/274/files#diff-e42d0ee91ea0ff69218f82b15dd993334ddcfa42f9244af166637f74f22e78daR571

I also can't see the disadvantages on doing this and we already have vue components with similar behavior.